cloudfour/cloudfour.com-patterns
# Add Gutenberg Stories

## Overview

This PR adds a single stories file containing un-styled stories for readily theme-able Gutenberg blocks. These are blocks that generate markup with added `wp-block-` classes.

The examples aim to provide ample detail for understanding variations that may be generated using the block editor controls. However, it is not exhaustive, and the API for each block is not easily discoverable.  In almost all cases, the markup and classes added to examples is based on discovery using the editor on a local Wordpress installation.

## Screenshots

![image](https://user-images.githubusercontent.com/771447/78412810-9bbdb980-75c9-11ea-88f5-3eb526444345.png)

## Testing

1. View the [preview theme](https://deploy-preview-590--cloudfour-patterns.netlify.com/)
1. See [cloudfour.com-wp issue #450](https://github.com/cloudfour/cloudfour.com-wp/issues/450)
1. Ensure all non-widget core blocks with a 🎨 icon have a story
1. Provide feedback regarding copy, examples (easily updated), and any other suggestions

---

- Addresses  [cloudfour.com-wp issue #450](https://github.com/cloudfour/cloudfour.com-wp/issues/450)

/CC @tylersticka @derekshirk
